engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -529,13 +533,14 @@ public void reconnectInputToOutput(String fromNodeUri, int outputId, Node toNode
      */
     private void reconnectInputToOutput(Node toNode, int inputId, DependencyConnection fromConnection,
                                         ConnectionType connectionType, boolean disconnectPrevious) {
-        logger.debug("Attempting reconnection of {} to {}'s output.", toNode.getUri(), fromConnection.getParentNode());
+        logger.atDebug().addArgument(() -> toNode.getUri()).addArgument(() -> fromConnection.getParentNode()).
+                log("Attempting reconnection of {} to {}'s output.");
         Node fromNode;
 
         fromNode = findNode(fromConnection.getParentNode());
         if (!fromConnection.getConnectedConnections().isEmpty()) {
-            logger.warn("WARNING: destination connection ({}) is already connected to ({})",
-                    fromConnection, fromConnection.getConnectedConnections());
+            logger.atWarn().addArgument(fromConnection).addArgument(() -> fromConnection.getConnectedConnections()).
+                    log("WARNING: destination connection ({}) is already connected to ({})");
```

Comment:
simple getter method
```suggestion
            logger.warn("WARNING: destination connection ({}) is already connected to ({})",
                    fromConnection, fromConnection.getConnectedConnections()); //NOPMD
```